tests/test_commands.py

```diff
@@ -87,3 +87,32 @@ def test_create_item_no_fgdc(self):
             item = pystac.read_file(item_path)
 
         item.validate()
+
```

Comment:
add a negative test when there's something wrong with the metadata.

also a test to test the id and datetime extraction failure directly.